modules/cram/templates/spectre_call.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+set -euo pipefail
+
+mosdepth () {
+    local args=()
+    args+=("--threads"  "!{task.cpus}")
+    args+=("--by" "1000")
+    args+=("--fasta" "!{paramReference}")
+    args+=("--mapq" "20")
+    args+=("--fast-mode")
+    args+=("mosdepth")
+    args+=("!{cram}")
+    ${CMD_MOSDEPTH} "${args[@]}"
+}
+
+call_copy_number_variation () {
+    local args=()
+    args+=("CNVCaller")
+    args+=("--coverage" "mosdepth.regions.bed.gz")
+    args+=("--sample-id" "!{sampleId}")
+    args+=("--output-dir" "spectre")
+    args+=("--reference" "!{paramReference}")
+    args+=("--metadata" "!{paramMetadata}")
+    args+=("--blacklist" "!{paramBlacklist}")
+    if [ "!{sampleSex}" == "male" ]; then
```

Comment:
this assumes `female` if sample sex is unknown. is this the desired behavior? will it work for 'unknown' sample that is actually a male?